.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: clone the repo
+      run: git clone https://github.com/sumangala028/learn-github-action.git
+    - name: execute the react project
```

Comment:
Why do you want to execute here? Is it a web browser to execute?

Reply:
So, how to run and see the result of the project

Reply:
Do you actually need to run this? What are other options you have in scripts? (check package.json)

Reply:
I can't understand the concept behind it

Reply:
Do follow up on the below comment